src/app/issuer/components/badge-studio/badge-studio.component.ts

```diff
@@ -233,10 +233,12 @@ export class BadgeStudioComponent implements OnInit, OnChanges {
 							'data:image/svg+xml;charset=utf-8,' +
 							encodeURIComponent(new XMLSerializer().serializeToString(svgRoot));
 						// const svgDataUrl2 = 'data:image/svg+xml;charset=utf-8,' + encodeURIComponent(new XMLSerializer().serializeToString(svgRoot2));
-						const svgImage = new Image();
-						const svgImage2 = new Image();
+						let svgImage = new Image();
```

Comment:
Why is it necessary to make this mutable? You should be able to reassign properties / attributes whilst it being `const`

Reply:
I made it `constant` again.